### ISSUE TITLE
Video <iframe> on FAQ Page Broken

### ISSUE THREAD

**Issue by yvvijay121** [PolicyEngine/policyengine#587]
I don't know if this is related to the new landing page PR, or my browser settings accidently breaking the page.

![image](https://user-images.githubusercontent.com/62124499/160293587-213d7a27-d364-4ba1-8361-0ed74bb75b17.png)

**Comment by nikhilwoodruff** [comment]
Yeah, after a few attempts I couldn't get this to work (thanks for filing). I'll remove it for now
